lib/log_hog/handler.ex

```diff
@@ -1,31 +1,41 @@
 defmodule LogHog.Handler do
   @moduledoc """
-  [`logger handler`](https://www.erlang.org/doc/apps/kernel/logger_chapter.html#handlers)
+  A [`logger handler`](https://www.erlang.org/doc/apps/kernel/logger_chapter.html#handlers).
   """
   @behaviour :logger_handler
 
   alias LogHog.Context
 
   @impl :logger_handler
   def log(log_event, %{config: config}) do
-    cond do
-      get_in(log_event, [:meta, :crash_reason]) ->
-        event = to_event(log_event, config)
-        LogHog.Sender.send(event, config.supervisor_name)
+    maybe_properties =
+      cond do
+        get_in(log_event, [:meta, :crash_reason]) ->
+          properties(log_event, config)
 
-      is_nil(config.capture_level) ->
-        :ok
+        is_nil(config.capture_level) ->
+          nil
 
-      Logger.compare_levels(log_event.level, config.capture_level) in [:gt, :eq] ->
-        event = to_event(log_event, config)
-        LogHog.Sender.send(event, config.supervisor_name)
+        Logger.compare_levels(log_event.level, config.capture_level) in [:gt, :eq] ->
+          properties(log_event, config)
 
-      true ->
-        :ok
+        true ->
+          nil
+      end
+
+    with %{} = properties <- maybe_properties do
+      LogHog.bare_capture(
```

Comment:
Why aren't we including the context here and using `bare_capture` rather than `capture`?

Reply:
Yeah, this makes perfect sense, thank you